lib/emit.generated.js

```diff
@@ -111,6 +111,71 @@ function getInt32Memory0() {
   return cachegetInt32Memory0;
 }
 
+function debugString(val) {
```

Comment:
Is it here on purpose?

Reply:
I guess the latest wasm bindgen started including this.